generator/actions_core.ml

```diff
@@ -6616,6 +6616,24 @@ Restore the C<backupfile> (from a previous call to
 C<guestfs_ntfsclone_out>) to C<device>, overwriting
 any existing contents of this device." };
 
+  { defaults with
+    name = "ntfs_chmod"; added = (1, 55, 13);
+    (* We don't use Pathname for `path` argument, since that implied
+       NEED_ROOT, and this command only operates on unmounted devices.
+    *)
+    style = RErr, [String (Device, "device"); String (PlainString, "mask"); String (PlainString, "path")], [OBool "recursive"];
```

Comment:
Is `mask` actually a number?  Can it ever be just text?  If it's always a number it'd be better to model it as an `Int` or `Int64` or similar.